### PR TITLE
Convert nautical miles to km

### PR DESCRIPTION
**Pull request by seandgrimes**
Converts nautical miles to kilometers, looking for either
"x nautical miles" or "x nmi" in a reddit post in order
to trigger the conversion

### CODE REVIEW

File: src/conversion_helper.js [hunk 1]
```diff
@@ -314,6 +314,14 @@ const unitLookupList = [
       }
     },
     "ignoredUnits" : [/° ?C/, "degrees? c", "celsius", "kelvin"]
+  },
+  {
+    "imperialUnits" : [/nmi/, /nautical\smiles?/],
+    "standardInputUnit" : " nmi",
+    "isInvalidInput" : isZeroOrNegative,
+    "isWeaklyInvalidInput": isHyperbole,
+    "conversionFunction" : (i) => createMap(i * 1.852, "km"),
```
Comment (cannawen) [inline src/conversion_helper.js:323]: There is a helper function in this file called `distanceMap()` that takes in a distance in meters and returns a map with an appropriate SI unit

**Comment by cannawen** [comment]
Silly technicality, but it's still September and I think the PRs have to be submitted in October for them to count towards the Hacktoberfest swag. Do you want to re-submit this PR then?

**Comment by seandgrimes** [comment]
Total brain fart moment, but yeah I'll just go ahead and resubmit the PR in a few days when Hacktoberfest actually starts :)

File: src/conversion_helper.js (hunk 1 above)
Comment (seandgrimes) [reply src/conversion_helper.js:323]: Cool beans, thanks for pointing that out.

**Pull request closed without merging**